sklearn/model_selection/_split.py

```diff
@@ -997,6 +997,19 @@ def split(self, X, y=None, groups=None):
             for train_index, test_index in cv.split(X, y, groups):
                 yield train_index, test_index
 
+    def get_n_splits(self, X=None, y=None, groups=None):
+        """Returns the number of splitting iterations in the cross-validator
```

Comment:
Can you document all the parameters of this function. You probably can copy and paste the documentation of another `get_n_splits` in this file.